src/app/cras/edit-cra/invoice-form/invoice-form.component.html

```diff
@@ -0,0 +1,314 @@
+    </div>
+    <div class="invoice-form__block__input-wrapper invoice-form__block__input-wrapper--has-next-line">
+      <label class="invoice-form__block__input-wrapper__item__input__toggle">
+        <input
+        type="checkbox"
+        name="invoicePaymentLatePenaltyToggle"
+        formControlName="invoicePaymentLatePenaltyToggle"
+        >
+        Pénalités de retard légales (40€ par facture + 10% par an)
+      </label>
+    </div>
+    <div class="invoice-form__block__input-wrapper">
+        <div class="invoice-form__block__input-wrapper__sub-block">
+          <h4 class="invoice-form__block__input-wrapper__sub-block__title">Coordonnées bancaires</h4>
+          <div class="invoice-form__block__input-wrapper__sub-block__input-wrapper invoice-form__block__input-wrapper__sub-block__input-wrapper--has-next-line">
+            <div class="invoice-form__block__input-wrapper__sub-block__input-wrapper__item form-group">
+                <label for="invoiceBankDetailsIBANInput" class="invoice-form__block__input-wrapper__sub-block__input-wrapper__item__label">IBAN</label>
+                <div class="invoice-form__block__input-wrapper__sub-block__input-wrapper__item__input">
+                    <input
+                    type="text"
+                    class="form-control"
+                    name="invoiceBankDetailsIBANInput"
+                    formControlName="invoiceBankDetailsIBANInput"
+                    >
+                </div>
+            </div>
+          </div>
+          <div class="invoice-form__block__input-wrapper__sub-block__input-wrapper">
+              <div class="invoice-form__block__input-wrapper__item invoice-form__block__input-wrapper__item--left form-group" >
+                  <label for="invoiceBankDetailsSWIFTInput" class="invoice-form__block__input-wrapper__item__label">SWIFT</label>
+                  <div class="invoice-form__block__input-wrapper__item__input">
+                      <input
+                      type="text"
+                      class="form-control"
+                      name="invoiceBankDetailsSWIFTInput"
+                      formControlName="invoiceBankDetailsSWIFTInput"
+                      >
+                  </div>
+              </div>
+              <div class="invoice-form__block__input-wrapper__item invoice-form__block__input-wrapper__item--right form-group" >
+                  <label for="invoiceBankDetailsDomiciliationInput" class="invoice-form__block__input-wrapper__item__label">Domiciliation</label>
+                  <div class="invoice-form__block__input-wrapper__item__input">
+                      <input
+                      type="text"
+                      class="form-control"
+                      name="invoiceBankDetailsDomiciliationInput"
+                      formControlName="invoiceBankDetailsDomiciliationInput"
+                      >
+                  </div>
+              </div>
+          </div>
+        </div>
+    </div>
+  </div>
+</div>
```

Comment:
Templates are getting big; Shouldn't we use directives to make them more readable?